fe/fe-core/src/main/java/org/apache/doris/qe/QeProcessorImpl.java

```diff
@@ -194,6 +194,7 @@ public TReportExecStatusResult reportExecStatus(TReportExecStatusParams params,
         }
         try {
             info.getCoord().updateFragmentExecStatus(params);
+            info.getCoord().setIsReportExecStatus(true);
```

Comment:
why add this code?

Reply:
> why add this code?



> why add this code?

The purpose is to determine whether FE has updated the status of BE execution so that streamload_action can query the execution status